### DISCUSSION TITLE
Anybody tried to run this library in a Chromium based browser

### DISCUSSION THREAD

**Discussion by hvician** [stoqey/ib#207] [Q&A]
I wrote a little client using the IBKR Web API. The issue with this is that I cannot be logged into the TWS and the Web API at the same time. I found this library and I love the TS doc seems to very promising. My question is did anybody try to run the library in a Chromium based browser using [Browserify](https://browserify.org)? I'd like to build a React or Svelte browser app to pull options data and I'm not sure what would be the best scalable approach. Maybe a Express.js server with a UI client or just a UI client using this library.

**Comment by tegryan-ddo** [comment]
I am running an app in Electron (https://www.electronjs.org/) and so far it's going OK, although I've just started

**Comment by ceddybi** [comment]
i usually set up a small server that maintains the connection for me in the background, then expose methods to the frontend ui, e.g rest end points and websockets for real-time data updates ... e.t.c 

checkout this wrapper [@stoqey/ibkr](https://github.com/stoqey/ibkr) for examples
